includes/User.php

```diff
@@ -2148,34 +2087,40 @@ public static function clearUserCache( $userId ) {
 	 *
 	 * @return bool
 	 */
-	public function setPassword( $str, $forceLogout=true ) {
+	public function setPassword( $str, $forceLogout = true ) {
 		global $wgAuth;
 
-		if( $str !== null ) {
-			if( !$wgAuth->allowPasswordChange() ) {
+		if ( $str !== null ) {
+			if ( !$wgAuth->allowPasswordChange() ) {
 				throw new PasswordError( wfMsg( 'password-change-forbidden' ) );
 			}
 
-			if( !$this->isValidPassword( $str ) ) {
+			if ( !$this->isValidPassword( $str ) ) {
 				global $wgMinimalPasswordLength;
 				$valid = $this->getPasswordValidity( $str );
 				if ( is_array( $valid ) ) {
 					$message = array_shift( $valid );
 					$params = $valid;
 				} else {
 					$message = $valid;
-					$params = array( $wgMinimalPasswordLength );
+					$params = [ $wgMinimalPasswordLength ];
 				}
-				throw new PasswordError( wfMsgExt( $message, array( 'parsemag' ), $params ) );
+				throw new PasswordError( wfMsgExt( $message, [ 'parsemag' ], $params ) );
 			}
 		}
 
-		if( !$wgAuth->setPassword( $this, $str ) ) {
+//		$return = self::heliosClient()->setPassword( $this->getId() );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)